.github/actions/setup-cli/action.yaml

```diff
@@ -22,14 +51,7 @@ runs:
 
     - id: setup-minikube
       run: |
-        ./hack/start-minikube.sh start --kubernetes-version=v${K8S_VERSION} --driver=$CONTAINER_RUNTIME --cpus $(nproc) --memory 5g
-      shell: bash
-
-    - id: docker-info
-      run: |
-        eval $(minikube docker-env)
```

Comment:
this line configured usage of internal minikube docker engine, not sure if we still want to use it.

Reply:
Right, we do not use that any more... thx, @pedjak